modules/log6x-quickstart-viaq.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
```

Comment:
The two separate quickstart files seem extremely unnecessary to me.    It's actually confusing to me that these two sections/files contain exactly the same resources.     The only difference is one annotation and one field value flipped from 'Viaq' to 'Otel'.    In my opinion it makes more sense to have one quickstart file, with one set of specs for UI, CLF, LokiStack, then include a section on the differences, with the details of  OTEL explained.

Reply:
This will be taken up in https://issues.redhat.com/browse/OBSDOCS-1494 .